src/m_vector.h

```diff
@@ -22,178 +22,186 @@
 
 #include "doomtype.h"
 
+#ifdef __clang__
```

Comment:
"Query for this feature with `__has_attribute(ext_vector_type)`"

https://clang.llvm.org/docs/LanguageExtensions.html